ocamldoc/.depend

```diff
@@ -276,3 +276,117 @@ generators/odoc_todo.cmx : odoc_module.cmx odoc_info.cmx odoc_html.cmx \
+    stdlib_non_prefixed/location.cmi stdlib_non_prefixed/lexing.cmi \
+    stdlib_non_prefixed/format.cmi
+stdlib_non_prefixed/lexing.cmi :
+stdlib_non_prefixed/list.cmi :
+stdlib_non_prefixed/listLabels.cmi :
+stdlib_non_prefixed/location.cmi : stdlib_non_prefixed/warnings.cmi \
+    stdlib_non_prefixed/lexing.cmi stdlib_non_prefixed/format.cmi
+stdlib_non_prefixed/longident.cmi :
+stdlib_non_prefixed/map.cmi :
+stdlib_non_prefixed/marshal.cmi :
+stdlib_non_prefixed/moreLabels.cmi : stdlib_non_prefixed/set.cmi \
+    stdlib_non_prefixed/map.cmi stdlib_non_prefixed/hashtbl.cmi
+stdlib_non_prefixed/nat.cmi :
+stdlib_non_prefixed/nativeint.cmi :
+stdlib_non_prefixed/num.cmi : stdlib_non_prefixed/ratio.cmi \
+    stdlib_non_prefixed/nat.cmi stdlib_non_prefixed/big_int.cmi
+stdlib_non_prefixed/obj.cmi : stdlib_non_prefixed/int32.cmi
+stdlib_non_prefixed/oo.cmi : stdlib_non_prefixed/camlinternalOO.cmi
+stdlib_non_prefixed/parse.cmi : stdlib_non_prefixed/parsetree.cmi \
+    stdlib_non_prefixed/lexing.cmi
+stdlib_non_prefixed/parser.cmi : stdlib_non_prefixed/parsetree.cmi \
+    stdlib_non_prefixed/location.cmi stdlib_non_prefixed/lexing.cmi \
+    stdlib_non_prefixed/docstrings.cmi
+stdlib_non_prefixed/parsetree.cmi : stdlib_non_prefixed/longident.cmi \
+    stdlib_non_prefixed/location.cmi stdlib_non_prefixed/asttypes.cmi
+stdlib_non_prefixed/parsing.cmi : stdlib_non_prefixed/obj.cmi \
+    stdlib_non_prefixed/lexing.cmi
+stdlib_non_prefixed/pervasives.cmi : \
+    stdlib_non_prefixed/camlinternalFormatBasics.cmi
+stdlib_non_prefixed/pprintast.cmi : stdlib_non_prefixed/parsetree.cmi \
+    stdlib_non_prefixed/format.cmi
+stdlib_non_prefixed/printast.cmi : stdlib_non_prefixed/parsetree.cmi \
+    stdlib_non_prefixed/format.cmi
+stdlib_non_prefixed/printexc.cmi :
+stdlib_non_prefixed/printf.cmi : stdlib_non_prefixed/buffer.cmi
+stdlib_non_prefixed/queue.cmi :
+stdlib_non_prefixed/random.cmi : stdlib_non_prefixed/nativeint.cmi \
+    stdlib_non_prefixed/int64.cmi stdlib_non_prefixed/int32.cmi
+stdlib_non_prefixed/ratio.cmi : stdlib_non_prefixed/nat.cmi \
+    stdlib_non_prefixed/big_int.cmi
+stdlib_non_prefixed/scanf.cmi : stdlib_non_prefixed/pervasives.cmi
+stdlib_non_prefixed/set.cmi :
+stdlib_non_prefixed/sort.cmi :
+stdlib_non_prefixed/spacetime.cmi :
+stdlib_non_prefixed/stack.cmi :
+stdlib_non_prefixed/stdLabels.cmi : stdlib_non_prefixed/stringLabels.cmi \
+    stdlib_non_prefixed/listLabels.cmi stdlib_non_prefixed/bytesLabels.cmi \
+    stdlib_non_prefixed/arrayLabels.cmi
+stdlib_non_prefixed/str.cmi :
+stdlib_non_prefixed/stream.cmi :
+stdlib_non_prefixed/string.cmi :
+stdlib_non_prefixed/stringLabels.cmi :
+stdlib_non_prefixed/syntaxerr.cmi : stdlib_non_prefixed/location.cmi \
+    stdlib_non_prefixed/format.cmi
+stdlib_non_prefixed/sys.cmi :
+stdlib_non_prefixed/uchar.cmi :
+stdlib_non_prefixed/unix.cmi : stdlib_non_prefixed/camlinternalBigarray.cmi
+stdlib_non_prefixed/warnings.cmi :
```

Comment:
These lines look stale. They are now in ocamldoc/stdlib_non_prefixed/.depend which looks right